content/docs/pipelines/overview/interfaces.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Introduction to the Pipelines Interfaces"
+description = "The ways you can interact with the Kubeflow Pipelines system"
+weight = 20
++++
+
+This page introduces the interfaces that you can use to build and run
+machine learning (ML) workflows with Kubeflow Pipelines.
+
+## User interface (UI)
+
+You can access the Kubeflow Pipelines UI by clicking **Pipeline Dashboard** on 
+the Kubeflow UI. The Kubeflow Pipelines UI looks like this:
+  <img src="/docs/images/pipelines-ui.png" 
+    alt="Pipelines UI"
+    class="mt-3 mb-3 border border-info rounded">
+
+From the Kubeflow Pipelines UI you can perform the following tasks:
+
+* Run one or more of the preloaded samples to try out pipelines quickly.
+* Upload a pipeline as a zipped YAML file. The pipeline can be one that you
```

Comment:
i would suggest skip mentioning the file format inside the zip package, since it's implementation detail. also recommend adding a link to the instruction on how the zip package is built, if exist.

Reply:
Done.